tools/launch_ccm_cluster.py

```diff
@@ -342,7 +342,7 @@ def __init__(
             start_timeout_mins = CCMLauncher.DEFAULT_TIMEOUT_MINS,
             public_agents = 0,
             private_agents = 1,
-            aws_region = 'eu-central-1',
+            aws_region = 'us-west-2',
```

Comment:
We have more resources here, CCM was failing, no one seems to know why we ever used eu-central-1.  Can remove though.